packages/react-sdk/src/state/communication/matrixRtcCommunicationChannel.ts

```diff
@@ -56,62 +62,40 @@ export class MatrixRtcCommunicationChannel implements CommunicationChannel {
   };
   private readonly peerConnections: PeerConnection[] = [];
   private sfuConfig: SFUConfig | undefined;
+  private activeFocus: RTCFocus | undefined;
```

Comment:
Do we need `activeFocus` field at all? I see we just read it from `sessionManager.getActiveFocus()`